src/cli/discord.ts

```diff
@@ -53,9 +53,28 @@ function filterEnv (env: NodeJS.ProcessEnv): NodeJS.ProcessEnv {
   return res
 }
 
-async function findDiscord (): Promise<string> {
-  // todo
-  return '/opt/discord-canary/DiscordCanary'
+async function findDiscord (): Promise<string | null> {
+  let path = null
```

Comment:
a habit + it gives me an error.

Reply:
I'd still prefer returns instead (& a final return null to match the types)

Reply:
removing the "let" will return a typescript error,
 and I need to make the variable out of the if statement to make it in the scope of the function.
 and for the types the variable is set to null unless the system matches the if statements then it will be changed to the 
 path for that system.
 for example if a macos user used Powerunit none of the if statements will be true so the returned value is null and 
 "Discord not found" error will be logged

Reply:
eh nevermind the comment, I'll refactor it myself later when I finalize this bit with the bits required for linux.